code/DefaultMembers.php

```diff
@@ -55,9 +63,9 @@ public static function create_default_members($silent=FALSE) {
                     $token = $member->generateAutologinTokenAndStoreHash();
                     $e = Member_ForgotPasswordEmail::create();
                     $e->populateTemplate($member);
-                    $e->populateTemplate(array(
-                      'PasswordResetLink' => Security::getPasswordResetLink($member, $token)
-                    ));
+                    $e->populateTemplate([
+                        'PasswordResetLink' => Security::getPasswordResetLink($member, $token),
+                    ]);
```

Comment:
Please update the minimum PHP version requirements in composer.json (defining arrays via `[ ]` is not supported by lower versions of PHP... not sure where the cutoff is).

Reply:
Updated composer.json